src/lib/geoip/client.ts

```diff
@@ -41,16 +44,20 @@ export default class GeoipClient {
 	async lookup (addr: string): Promise<ProbeLocation> {
 		const results = await Promise
 			.allSettled([
-				this.lookupWithCache<LocationInfo>(`geoip:ipinfo:${addr}`, async () => ipinfoLookup(addr)),
+				this.lookupWithCache<LocationInfo>(`geoip:ip2location:${addr}`, async () => ip2LocationLookup(addr)),
+				this.lookupWithCache<LocationInfo>(`geoip:ipmap:${addr}`, async () => ipmapLookup(addr)),
 				this.lookupWithCache<LocationInfo>(`geoip:maxmind:${addr}`, async () => maxmindLookup(addr)),
+				this.lookupWithCache<LocationInfo>(`geoip:ipinfo:${addr}`, async () => ipinfoLookup(addr)),
 				this.lookupWithCache<FastlyBundledResponse>(`geoip:fastly:${addr}`, async () => fastlyLookup(addr)),
 			])
-			.then(([ ipinfo, maxmind, fastly ]) => {
-				const fulfilled = [];
+			.then(([ ip2location, ipmap, maxmind, ipinfo, fastly ]) => {
```

Comment:
I see the order changes here and shifts ipinfo from the first position to the fourth. I understand it's based on the Google table, but with the radius matching (not yet implemented here), it seems ipinfo is again the best one - do you plan to then reorder this again?

Reply:
It depends on do we want to vote before the radius matching (using the original provider cities) or after (using the radius approximated cities). I was thinking about the first option [initialy](https://github.com/jsdelivr/globalping/issues/264#issuecomment-1604376222), but second makes sense too.

Nothing prevents us from implementing the approximation first and then take a look at the results. Reordering of steps as well as reprioritising of providers can be done easily.

Reply:
I asked because based on all our previous observations, ipinfo seemed to be the most precise one. The Google table currently shows something else but it still isn't a very big sample and ipinfo jumps back to the top again if we consider radiuses - so @jimaek and I still think it should have the highest priority.

Reply:
Best radius (60 km) was chosen based on the ipinfo data. Then that radius was applied to other providers, so approximation results may be a bit bound to ipinfo. But anyway if we are comparing by approximated cities priorities should be changed and ipinfo is the most accurate right now.